great_expectations/util.py

```diff
@@ -2152,3 +2153,84 @@ def numpy_quantile(
         )
 
     return quantile
+
+
+def add_dataframe_to_db(
```

Comment:
Helper method that is used to wrap `to_sql()` and suppress warnings.

Reply:
❤️

Reply:
Good idea @anthonyburdi. Doing now

Reply:
This is now in the `df_to_database_loader.py` file